src/molecules/select/Select.html

```diff
@@ -1,19 +1,22 @@
-<div class="select">
+<div class="a-select">
   <!-- @slot Label content (Named slot) -->
   <slot name="label">
```

Comment:
Why not scoped? :wink:

Reply:
Make sense, thanks ! :blush:

Reply:
I removed `label="Label text"`from story `With slots` - now should be ok :blush: